metacat-connector-polaris/src/main/java/com/netflix/metacat/connector/polaris/PolarisConnectorTableService.java

```diff
@@ -388,7 +391,10 @@ public List<QualifiedName> getTableNames(
             final List<QualifiedName> result = Lists.newArrayList();
             for (int i = 0; i < databaseNames.size() && limitSize > 0; i++) {
                 final String databaseName = databaseNames.get(i);
-                final List<String> tableNames = polarisStoreService.getTables(name.getDatabaseName(), "");
+                final List<String> tableNames = polarisStoreService.getTables(
+                    name.getDatabaseName(),
+                    "",
+                    1000);
```

Comment:
Since names are smaller than the objects themselves, it might be more efficient to make the constant larger for names, maybe we can try 5k or 10k

Reply:
Looks like this one was missed, can you update it to read from config as well